datashuttle/datashuttle.py

```diff
@@ -168,6 +168,7 @@ def create_folders(
         sub_names: Union[str, List[str]],
         ses_names: Optional[Union[str, List[str]]] = None,
         datatype: str = "",
+        bypass_validation: bool = False,
```

Comment:
also needs to be added to the docstring.

Reply:
https://github.com/neuroinformatics-unit/datashuttle/pull/345#discussion_r1550613548